README.md

```diff
@@ -32,6 +32,16 @@ further details.
 
 It's worth pointing out that packages such as [Images.jl](https://github.com/JuliaImages/Images.jl) load FileIO for you.
 
+Loading an image is then as simple as
+
+```julia
+img = load(filename[; view=false])
+```
+
+The optional `view` keyword argument is useful for reducing memory consumption
+when loading large files.
```

Comment:
Not that you should feel responsible for doing this, simply asking the question is useful.